src/Definition/Entity.php

```diff
@@ -125,6 +134,9 @@ public function setScope(?string $scope): self
         return $this;
     }
 
+    /**
+     * @return class-string|null
```

Comment:
Shouldn't it be some `class-string<ScopeInterface>|null`?

Reply:
Added class-string<ScopeInterface>|null